### DISCUSSION TITLE
Feature Request: Drag and drop

### DISCUSSION THREAD

**Discussion by gcasu37** [cboxdoerfer/fsearch#577] [Ideas]
it would be very nice to have drag and drop functionality for example suppose you have an instance of kate or any other software that supports drag and drop from dolphin (i use qgis for example). it would be very nice to be able to open the file.txt directly on the instance you are working on instead of having to open a new dolphin window and then drag and drop from there into the file.

i use these functionality in windows all the time and they speedup my work a lot

**Comment by ts-kontakt** [comment]
I would really pay for this functionality.
(plus rename functionality)
